src/UsgsAstroFramePlugin.cpp

```diff
@@ -277,207 +273,199 @@ csm::Model *UsgsAstroFramePlugin::constructModelFromISD(const csm::Isd &imageSup
+  sensorModel->m_odtY[8] = atof(imageSupportData.param("optical_distortion_y", 8).c_str());
+  sensorModel->m_odtY[9] = atof(imageSupportData.param("optical_distortion_y", 9).c_str());
+
+
+  // sensorModel->m_ccdCenter[0] = atof(imageSupportData.param("ccd_center", 0).c_str());
+  // sensorModel->m_ccdCenter[1] = atof(imageSupportData.param("ccd_center", 1).c_str());
+  //
+  // sensorModel->m_originalHalfLines = atof(imageSupportData.param("original_half_lines").c_str());
+  // sensorModel->m_spacecraftName = imageSupportData.param("spacecraft_name");
+  //
+  // sensorModel->m_pixelPitch = atof(imageSupportData.param("pixel_pitch").c_str());
+  //
+  // sensorModel->m_iTransS[0] = atof(imageSupportData.param("itrans_sample", 0).c_str());
+  // sensorModel->m_iTransS[1] = atof(imageSupportData.param("itrans_sample", 1).c_str());
+  // sensorModel->m_iTransS[2] = atof(imageSupportData.param("itrans_sample", 2).c_str());
+  //
+  // if (imageSupportData.param("itrans_sample", 0) == "") {
+  //   missingKeywords.push_back("itrans_sample missing first element");
+  // }
+  // else if (imageSupportData.param("itrans_sample", 1) == "") {
+  //   missingKeywords.push_back("itrans_sample missing second element");
+  // }
+  // else if (imageSupportData.param("itrans_sample", 2) == "") {
+  //   missingKeywords.push_back("itrans_sample missing third element");
+  // }
+
+  sensorModel->m_ephemerisTime = atof(imageSupportData.param("center_ephemeris_time").c_str());
+  if (imageSupportData.param("center_ephemeris_time") == "") {
+    missingKeywords.push_back("center_ephemeris_time");
+  }
+
+  // sensorModel->m_iTransL[0] = atof(imageSupportData.param("focal2pixel_lines", 0).c_str());
+  // sensorModel->m_iTransL[1] = atof(imageSupportData.param("focal2pixel_lines", 1).c_str());
+  // sensorModel->m_iTransL[2] = atof(imageSupportData.param("focal2pixel_lines", 2).c_str());
+  // if (imageSupportData.param("focal2pixel_lines", 0) == "") {
+  //   missingKeywords.push_back("itrans_line needs 3 elements");
+  // }
+  // else if (imageSupportData.param("focal2pixel_lines", 1) == "") {
+  //   missingKeywords.push_back("itrans_line needs 3 elements");
+  // }
+  // else if (imageSupportData.param("focal2pixel_lines", 2) == "") {
+  //   missingKeywords.push_back("itrans_line needs 3 elements");
+  // }
+
+  sensorModel->m_nLines = atoi(imageSupportData.param("image_lines").c_str());
+  sensorModel->m_nSamples = atoi(imageSupportData.param("image_samples").c_str());
+  if (imageSupportData.param("image_lines") == "") {
+    missingKeywords.push_back("image_lines");
+  }
+  if (imageSupportData.param("image_samples") == "") {
+    missingKeywords.push_back("image_samples");
+  }
+
+  sensorModel->m_transY[0] = atof(imageSupportData.param("focal2pixel_lines", 0).c_str());
+  sensorModel->m_transY[1] = atof(imageSupportData.param("focal2pixel_lines", 1).c_str());
+  sensorModel->m_transY[2] = atof(imageSupportData.param("focal2pixel_lines", 2).c_str());
```

Comment:
m_transY is the coefficients for the affine from pixels to focal plane y. The focal2pixel parameters should be the inverse of that. We'll have to modify the camera model to make this work correctly. Right now, we have the wrong parameters in `focal2pixel_*` so it's not causing an error, but both should be fixed.